lib/phoenix_live_view/helpers.ex

```diff
@@ -767,20 +767,21 @@ defmodule Phoenix.LiveView.Helpers do
   ## Examples
 
       <%= for entry <- @uploads.avatar.entries do %>
-        <%= live_img_preview entry, width: 75 %>
+        <%= live_img_preview entry, class: "figure" %>
```

Comment:
Since the `img` tag is rendered by the hook we probably want to add a small snippet of CSS to the example:

```css
.figure {
  img {
    width: 75px;
  }
}
```